pkg/k8s/deployments/translate.go

```diff
@@ -37,8 +37,8 @@ const (
 	revisionAnnotation         = "deployment.kubernetes.io/revision"
 	//OktetoBinName name of the okteto bin init container
 	OktetoBinName = "okteto-bin"
-	//OktetoInitFromImage name of the okteto init from image container
-	OktetoInitDataName = "okteto-init-data"
+	//OktetoInitVolumeName name of the okteto init container that initializes the persistent colume from image content
+	OktetoInitVolumeName = "okteto-init-volume"
```

Comment:
if it's already a volume, do you think it might be redundant to call it `-volume`?